cou/apps/base.py

```diff
@@ -513,13 +514,21 @@ def _get_upgrade_current_release_packages_plan(self, parallel: bool = False) ->
         :return: Plan for upgrading software packages to the latest of the current release.
         :rtype: PreUpgradeStep
         """
-        return PreUpgradeStep(
+        step = PreUpgradeStep(
```

Comment:
Should we change this to a plan instead of step? It have empty coro and allows its substeps to be run in parallel. I'm pointing this out because in #235 we are removing all `parallel` param in step functions, and we've been trying to draw a clear line between plan and step.

Reply:
After second thought, I do not see reason to make it as plan, since this is a sub-step of application plan. I agree that it does not have coro, but we want to make it parallel = True and plan do not have such option. Do you think we should add PreUpgradePlan? I think that it's not needed.